audience-analytics/index.rst

```diff
@@ -0,0 +1,289 @@
+.. _audience-tuto:
+
+Audience Analytics
+==================
+
+|
+
+Step 1 - Introduction
+*********************
+
+This documentation is meant at developers wanting to install, configure and launch Angus.ai audience analytics application on a personal computer.
```

Comment:
Lack of homogeneity on term (I think it is the same things): device, computer, player

Reply:
Fixed